src/Components/Blazor/Server/src/MonoDebugProxy/BlazorMonoDebugProxyAppBuilderExtensions.cs

```diff
@@ -122,8 +122,8 @@ private static async Task DebugHome(HttpContext context)
                 await context.Response.WriteAsync($@"
                     <h1>Unable to find debuggable browser tab</h1>
                     <p>
-                        The response from <code>{debuggerTabsListUrl}</code> does not include
-                        any entry for <code>{targetTabUrl}</code>.
+                        The response from <c>{debuggerTabsListUrl}</c> does not include
```

Comment:
Sorry, I wasn't clear earlier. The entire file is incorrect. Can you do a pass over all the changes and make sure all the changed tags are **inside XML comments**?

Reply:
just updated the code.